main.cc

```diff
@@ -1 +1,12 @@
 #include <iostream>
+#include <stdlib.h>
+#include <time.h> 
+
+int main(void)
+{
+srand((unsigned)time(NULL)); 
+for(int i = 0; i < 100;i++ ) 
```

Comment:
```suggestion
for(int i = 0; i < 100; ++i) 
```
Always preincreament